resources/rickroll

```diff
@@ -1 +1,67 @@
+b"We're no strangers to love
```

Comment:
Can you please move this file to `src/`? I think making a `resources/` directory with one file is superfluous.